### ISSUE TITLE
Error when calling Hijri::Date.new(1439, 4, 12).to_greo

### ISSUE THREAD

**Issue by abahaggag** [ecleel/hijri#5]
Hijri gem version: 0.5.0
I got this error when calling to_greo method on Hijri::Date with this date (1439, 4, 12)

```
Hijri::Date.new(1439, 4, 12).to_greo

NoMethodError: undefined method `+' for nil:NilClass
from /home/ahmed/.rvm/gems/ruby-2.2.3@arbab/gems/hijri-0.5.0/lib/hijri/converter.rb:54:in `block in greo_to_absolute'
```

I have no idea about conversions and why it gives error for this date or maybe for other dates with similar patterns. Please check and let me know the issue

**Comment by ecleel** [comment]
Hello @abahaggag 

Sorry, it took me ages to fix the issue but it's fixed now and pushed to the master and later on will release a new version with all fixes.

**Comment by ecleel** [comment]
the issue has been resolved in v0.6.0

**Comment by abahaggag** [comment]
Thanks @ecleel
